21/ex21.py

```diff
@@ -0,0 +1,59 @@
+#this function adds two arguments passed to the function. The output for one prints the action of the function
+#the second is htat it returns the sum of the two variables. 
+#When this function is assigned to a variable, the outcome of the return actio will be assigned to the variable.n 
+def add(a, b):
+	print "ADDING %d + %d" % (a, b)
+	return a + b
+#subtracts the two arguments passed on to the function
+def subtract(a, b):
+	print "SUBTRACTING %d - %d" % (a, b)
+	return a - b
+#multiply two arguments passed to the fucntion.
+def multiply(a, b):
+	print "MULTIPLYING %d * %d" % (a, b)
+	return a * b
+#divides the two arguments passed ot the function
+def divide(a, b):
+	print "DIVIDING %d / %d" % (a, b)
+	return a / b
+
+print "Let's do some math with just functions!"
+#we assign the outcoms of the above functions to variables so that they can be used in additional functions.
+age = add(40,5)
+height = subtract(80,4)
+weight = multiply(90,2)
+iq = divide(110,2)Computer2241
```

Comment:
Didn't even notice that, had to type that somewhere else was probably still
stuck on this screen

On Wed, Feb 24, 2016 at 5:21 PM, Arjun Balaji <notifications@github.com>
wrote:

> In 21/ex21.py
> <https://github.com/manitghogar/lpthw/pull/5#discussion_r54017180>:
>
> > +	return a - b
> > +#multiply two arguments passed to the fucntion.
> > +def multiply(a, b):
> > +	print "MULTIPLYING %d * %d" % (a, b)
> > +	return a * b
> > +#divides the two arguments passed ot the function
> > +def divide(a, b):
> > +	print "DIVIDING %d / %d" % (a, b)
> > +	return a / b
> > +
> > +print "Let's do some math with just functions!"
> > +#we assign the outcoms of the above functions to variables so that they can be used in additional functions.
> > +age = add(40,5)
> > +height = subtract(80,4)
> > +weight = multiply(90,2)
> > +iq = divide(110,2)Computer2241
>
> What is Computer2241?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/manitghogar/lpthw/pull/5/files#r54017180>.
>